util.go

```diff
@@ -35,6 +35,17 @@ import (
 	"github.com/urfave/cli"
 )
 
+const (
+	vendorConf = "vendor.conf"
+	modulesTxt = "vendor/modules.txt"
+	goMod      = "go.mod"
+)
+
+var (
+	ErrUnknownFormat = errors.New("unknown file format")
```

Comment:
Can you un-export these as `errUnknownFormat`. If we add linting here (we probably should) it might complain about the lack of comments.